clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -33,26 +36,38 @@ const ALL_COLUMNS: ColumnMetadata[] = [
 const ScanResultsForm = () => {
   const systems = useAppSelector(selectSystemsForReview);
   const dispatch = useAppDispatch();
+  const router = useRouter();
   const {
     isOpen: isWarningOpen,
     onOpen: onWarningOpen,
     onClose: onWarningClose,
   } = useDisclosure();
+  const [upsertSystems] = useUpsertSystemsMutation();
   const [selectedSystems, setSelectedSystems] = useState<System[]>(systems);
+  const features = useFeatures();
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
 
+  const createSystems = async () => {
+    await upsertSystems(selectedSystems);
+  };
+
   const confirmRegisterSelectedSystems = () => {
     dispatch(chooseSystemsForReview(selectedSystems.map((s) => s.fides_key)));
-    dispatch(changeStep());
+    createSystems();
+
+    if (features.plus) {
+      router.push(`/datamap`);
```

Comment:
it just occurred to me that this won't work, since the datamap UI will be in a different nextjs zone. I think you'll have to do something like what Sebastian implemented for the navbar. and it'll be a pain to test 😬 

https://github.com/ethyca/fides/blob/1479/systems-without-declarations-default/clients/admin-ui/src/features/common/nav/NavLink.tsx/#L29-L32